wordpress-mobile/WordPress-iOS
# New Domains Dashboard Card UI

This PR adds functionality for the new Domains Dashboard Card UI as part of the Domains Dashboard Card project

Note that the card tap actions such as onViewTap action have not been implemented and the area to implement has been marked with a TODO

Fixes #20409

To test:
  1. Launch the iOS Jetpack app
  2. After logging in choose a site
  3. Domain Dashboard card should be visible

## Regression Notes
1. Potential unintended areas of impact
Dashboard

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual tests

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:
